en/docs/develop/product-apis/admin-apis/admin-v1/admin-v1.yaml

```diff
@@ -262,6 +279,20 @@ paths:
             Location header contains URL of newly created entity.
           schema:
             $ref: '#/definitions/ApplicationThrottlePolicy'
+          examples:
+            application/json:
```

Comment:
An example is already defined for ApplicationThrottlePolicy. Why do we have to repeat an example here?

Reply:
Since ApplicationThrottlePolicy is an extension of ThrottlePolicy when fetching the schema level example, it always displays the generic example values given in ThrottlePolicy definition. Therefore had to add a response level example(which has higher precedence than the schema level example) to override the generic example in a meaningful way.